src/Sylius/Behat/Page/Shop/Account/Order/IndexPage.php

```diff
@@ -104,4 +105,61 @@ protected function getDefinedElements()
             'last_order' => 'table tbody tr:last-child a:contains("Show")',
         ]);
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function isCancelButtonVisibleForOrderWithNumber(string $number): bool
+    {
+        try {
+            $orderData = $this->getOrderData($number);
+        }  catch (\Exception $exception) {
+            return false;
+        }
+
+        $actionButtonsText = $orderData->find('css', 'td:last-child')->getText();
+
+        return strpos($actionButtonsText, 'Cancel');
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function clickCancelButtonNextToTheOrder(string $number): void
+    {
+        $orderData = $this->getOrderData($number);
+
+        $cancelButton = $orderData->find('css', 'td:last-child button:contains("Cancel")');
+
+        if (null === $cancelButton) {
+            throw new \Exception(sprintf('There is no cancel button next to order %s', $number));
```

Comment:
Maybe we should simplify it to `Assert::notNull($cancelButton, sprintf('There is no cancel button next to order %s', $number))`